src/main/osd/osd_elements.c

```diff
@@ -927,25 +930,25 @@ static void osdElementRateProfileName(osdElementParms_t *element)
 static void osdElementPidProfileName(osdElementParms_t *element)
 {
     if (strlen(currentPidProfile->profileName) == 0) {
-        tfp_sprintf(element->buff, "PID_%u", getCurrentPidProfileIndex() + 1);
+        tfp_sprintf(element->buff, "%s%u", STR_OSDE_PID, getCurrentPidProfileIndex() + 1);
     } else {
         toUpperCase(element->buff, currentPidProfile->profileName, MAX_PROFILE_NAME_LENGTH);
     }
 }
-#endif
+#endif // USE_PROFILE_NAMES
 
 #ifdef USE_OSD_PROFILES
 static void osdElementOsdProfileName(osdElementParms_t *element)
 {
     uint8_t profileIndex = getCurrentOsdProfileIndex();
 
     if (strlen(osdConfig()->profile[profileIndex - 1]) == 0) {
-        tfp_sprintf(element->buff, "OSD_%u", profileIndex);
+        tfp_sprintf(element->buff, "%s%u", STR_OSDE_OID, profileIndex);
```

Comment:
Something similar here, the constant "hides" the parameters, dificulting the reading. Maybe we can add some kind of validation when creating the language file to verify that the translation contains the same parameters than the original string, the translations sometimes have errors.

Reply:
I think we drop translation of those.
Its look ugly like STR_OSDE_OID_P_U, and maybe nobody understand thats mean OSD_%u
Agree ?

Reply:
now left a comment in bf_locale.xml, like "notice %u follow"